src/modules/jcms_notifications/jcms_notifications.install

```diff
@@ -0,0 +1,26 @@
+<?php
+
+function jcms_notifications_schema() {
+  $schema['jcms_notifications'] = [
+    'description' => 'The base table for nodes.',
+    'fields' => [
+      'notification_id' => [
+        'description' => 'The primary identifier for a notification.',
+        'type' => 'serial',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+      ],
+      'nid' => [
```

Comment:
@giorgiosironi We can call it `node_id` but I used `nid` because that's how Drupal refers to it everywhere else. Happy to amend it though.